src/virtual-machine.js

```diff
@@ -568,6 +584,28 @@ class VirtualMachine extends EventEmitter {
         }
     }
 
+    setStandaloneAliasVariables (data) {
+        if (typeof data.payload === 'string' && data.payload !== '') {
+            const stage = this.runtime.getTargetForStage();
+            // params (id, name, type, isCloud)
+            stage.createVariable(`${data.alias}`, `${data.alias}`, '', false);
+            setTimeout(() => {
+                stage.variables[data.alias].value = data.payload;
+            }, 100);
+        }
+    }
+
+    setStandaloneGroupVariables (data) {
```

Comment:
I think that what you want to do is set data to be in a standard format above and only have one function that does this stuff

Reply:
that won't work, the .createVariable method that is called without the `workspace` from the GUI is a different method with different params